roles/pulp_common/vars/main.yml

```diff
@@ -41,13 +43,15 @@ __pulp_common_pulp_settings_defaults:
   token_signature_algorithm: ES256
   working_directory: "{{ pulp_settings.deploy_root | default(pulp_user_home) | regex_replace('\\/$', '') }}/tmp"
   content_path_prefix: "{{ pulp_settings.content_path_prefix | default('/pulp/content/') }}"
+__pulp_common_pulp_settings_defaults_2:
+  file_upload_temp_dir: "{{ __pulp_common_pulp_settings_defaults_1.working_directory }}"
```

Comment:
I don't understand why you created this `__pulp_common_pulp_settings_defaults_2`

Reply:
Because I cannot reference __pulp_common_pulp_settings_defaults.working_directory from within __pulp_common_pulp_settings_defaults. Or at least, I assumed I couldn't.

I will test if I can.

Reply:
Tested, it fails if I try that with `An unhandled exception occurred while templating '{{ __pulp_common_pulp_settings_defaults`

Reply:
What would you suggest?

Reply:
sweet!